plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/GradleEnclavePlugin.kt

```diff
@@ -175,14 +175,17 @@ class GradleEnclavePlugin @Inject constructor(private val layout: ProjectLayout)
             enclaveMode: EnclaveMode,
             conclaveExtension: ConclaveExtension,
             enclaveFatJarTask: Jar,
-            signingKey: Provider<RegularFile?>
+            signingKey: Provider<RegularFile?>,
+            linuxExec: LinuxExec
     ): GenerateGramineBundle {
-        return target.createTask("generateGramine${enclaveMode.capitalise()}Bundle", enclaveMode) { task ->
+        return target.createTask("generateGramine${enclaveMode.capitalise()}Bundle", enclaveMode, linuxExec) { task ->
             // TODO: Build Gramine enclaves in conclave-build container: https://r3-cev.atlassian.net/browse/CON-1229
+            task.dependsOn(linuxExec)
             task.signingKey.set(signingKey)
             task.productId.set(conclaveExtension.productID)
             task.revocationLevel.set(conclaveExtension.revocationLevel)
             task.maxThreads.set(conclaveExtension.maxThreads)
+            task.dockerImageTag.set(linuxExec.tag)
```

Comment:
The tag to use must be the hash that we calculate over the containers directory. It can't be the CONCLAVE_SDK_VERSION as it's not specific enough.

Reply:
Yes, this is a temporary solution to make things running on my side. But still don't have clear, should be us to create the "hash" to identify the image? In this PR? If so, I suggest to use the list of package versions installed  (`dpkg`) + our files to do that, or is there any other thought on this?

Reply:
The tag is already calculated. It's the hash of the containers dir, and we do it here:
https://github.com/R3Conclave/conclave-core-sdk/blob/f26ba0b649cb663649568d12a70ad61169383d40/scripts/ci_build_common.sh#L29
The trick is to get this value and put it into the jar manifest. @filipesoliveira has the same task for his [ticket](https://r3-cev.atlassian.net/browse/CON-1069), so I suggest you two collaborate on the best solution.

Reply:
@bonifazi-r3, we can discuss the solution today. I have found a way to call scripts from Gradle and capture the output to a variable inside Gradle.